src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/Rfc7240ProxyIT.java

```diff
@@ -138,4 +138,33 @@ public void shouldHandleMultipleParallelRequestWithPreferWaitAndUpdatedAuthoriza
         counters.assertExpectedCacheEntries(1);
     }
 
+    @Test
+    @Specification({
+        "${route}/proxy/controller",
+        "${streams}/server.next.request.if.current.request.expired/accept/client",
+        "${streams}/server.next.request.if.current.request.expired/connect/server",
+    })
+    public void shouldServerNextRequestIfCurrentRequestExpired() throws Exception
```

Comment:
Does this mean to be `shouldServer` or `shouldServe`? Same question for script scenario name.

Reply:
Serve :)